base_layer/wallet/src/transaction_service/error.rs

```diff
@@ -40,6 +40,8 @@ use tokio::sync::broadcast::RecvError;
 pub enum TransactionServiceError {
     #[error("Transaction protocol is not in the correct state for this operation")]
     InvalidStateError,
+    #[error("One-sided tx error: `{0}`")]
```

Comment:
Personally I prefer to use the full word transaction.

Reply:
Better yes